app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -88,7 +88,7 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
                           (implicit hc: HeaderCarrier): Unit = {
     sendExplicitAuditEvent(
       auditEventType = RulingCancelled,
-      auditPayload = statusChangeAuditPayload(oldCase, updatedCase, operator)
+      auditPayload = statusChangeAuditPayload(oldCase, updatedCase, operator) // TODO: add reason for canceling ruling
```

Comment:
Do we put todos in code?   - in this case the reason for cancel is coming in a later story

Reply:
Not usually
I just wanted to add a reminder for you....